src/utils/categories.ts

```diff
@@ -1,9 +1,9 @@
-import MarketplaceItemModel from "@/models/marketplaceItemModel";
+import MarketplaceItemModel from "./../models/MarketplaceItemModel";
```

Comment:
Why to use relative path ant not continue using [name module mapper](https://github.com/Kentico/ems-extension-marketplace/blob/master/package.json#L88) ?